test/test_manifest.rb

```diff
@@ -610,7 +610,7 @@ class SlowExporter2 < SlowExporter
     @env.register_exporter 'image/png',SlowExporter
     @env.register_exporter 'image/png',SlowExporter2
     Sprockets::Manifest.new(@env, @dir).compile('logo.png', 'troll.png')
-    assert_equal %w(0 0 0 0 1 1 1 1), SlowExporter.seq
+    refute_equal %w(0 1 0 1 0 1 0 1), SlowExporter.seq
```

Comment:
Why are we asserting what this isn't instead of what it is?

Reply:
When we're exporting things serially then the second exporter will not start until the first one ends. We are guaranteed to get `0 1 0 1 0 `. However when we're running this in parallel, then we do not need to wait for the first exporter to be finished for the second to run, etc. That's why the original code asserted that we got four zeroes first. 

However that test is coupled to a `sleep` and therefore race conditions. 

I changed the test to allow for other combinations, as long as they don't perfectly match the "synchronous" case.